JuiceMaker/JuiceMaker/Model/Juice.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Juice.swift
+//  JuiceMaker
+//
+//  Created by Doogie & Malrang.
+//
+enum Juice {
+    case strawBerryJuice
+    case bananaJuice
+    case kiwiJuice
+    case pineappleJuice
+    case ddalBaJuice
+    case mangoJuice
+    case mangKiJuice
+    var recipie: [Fruits : Int] {
+        get {
+            switch self {
+            case .strawBerryJuice:
+                return [.strawberry : 16]
+            case .bananaJuice:
+                return [.banana : 2]
+            case .kiwiJuice:
+                return [.kiwi : 3]
+            case .pineappleJuice:
+                return [.pineapple : 2]
+            case .ddalBaJuice:
+                return [.strawberry : 10, .banana : 1]
+            case .mangoJuice:
+                return [.mango : 3]
+            case .mangKiJuice:
+                return [.mango : 2, .kiwi : 1]
+            }
+        }
```

Comment:
get을 작성해주셨네요:) set을 생략하고 get만 사용한다면 get을 제거해서 더 깔끔한 코드를 작성할 수 있을 것 같아요!
```suggestion
        switch self {
        case .strawBerryJuice:
            return [.strawberry : 16]
        case .bananaJuice:
            return [.banana : 2]
        case .kiwiJuice:
            return [.kiwi : 3]
        case .pineappleJuice:
            return [.pineapple : 2]
        case .ddalBaJuice:
            return [.strawberry : 10, .banana : 1]
        case .mangoJuice:
            return [.mango : 3]
        case .mangKiJuice:
            return [.mango : 2, .kiwi : 1]
        }
```

Reply:
이 부분은 말랑이 생략 가능하다고 말해주긴 했는데 제가 연산프로퍼티 사용이 처음이어서 헷갈려서 적어두자고 제안 했습니다!
프로젝트 최종본 제출시에는 생략해주도록 하겠습니다!